e2e/shadcn-ui-e2e/tests/shadcn-ui.spec.ts

```diff
@@ -22,14 +22,15 @@ describe('shadcn/ui e2e', () => {
       `${utilsLibName}/src/index.ts`,
       'components.json'
     )).not.toThrow()
-  })
 
-  // it('should be able add a button', async () => {
-  //   await runNxCommandAsync(`add ${uiLibName} button`)
-  //
-  //   expect(() => checkFilesExist(
-  //     `${uiLibName}/src/button.tsx`,
-  //   )).not.toThrow()
-  // })
```

Comment:
You think we can add this test? Prevents us from releasing a version again that suddenly stops working.